Library/Formula/tmux.rb

```diff
@@ -22,6 +22,10 @@ class Tmux < Formula
   depends_on "pkg-config" => :build
   depends_on "libevent"
 
+  # This fixes the Tmux 2.1 update that broke the ability to use select-pane [-LDUR]
+  # to switch panes when in a maximized pane https://github.com/tmux/tmux/issues/150#issuecomment-149466158
+  patch :DATA
```

Comment:
Since the patch has been committed we should only apply it to the stable build, and we can also use it directly:
```ruby
stable do
  url "https://github.com/tmux/tmux/releases/download/2.1/tmux-2.1.tar.gz"
  sha256 "31564e7bf4bcef2defb3cb34b9e596bd43a3937cad9e5438701a81a5a9af6176"

  patch do
    url "https://github.com/tmux/tmux/commit/a05c27a7e1c4d43709817d6746a510f16c960b4b.diff"
    sha256 "👻"
  end
end